test/system/roles/monitoree/assessment/form.rb

```diff
@@ -12,12 +12,36 @@ class AssessmentForm < ApplicationSystemTestCase
   PATIENTS = @@system_test_utils.patients
   ASSESSMENTS = @@system_test_utils.assessments
 
-  def complete_assessment(patient, assessment_label)
+  def complete_assessment(user_label, patient, assessment_label)
+    @@system_test_utils.login(user_label) unless user_label.nil?
     visit "/patients/#{patient.submission_token}/assessments/new"
     submit_assessment(ASSESSMENTS[assessment_label]['symptoms'])
     err_msg = @@system_test_utils.get_err_msg('Monitoree assessment', 'submission message', 'existent')
     assert page.has_content?('Thank You For Completing Your Self Report'), err_msg
-    @@assessment_form_verifier.verify_assessment(patient, ASSESSMENTS[assessment_label])
+    @@assessment_form_verifier.verify_assessment(patient, ASSESSMENTS[assessment_label]) unless user_label.nil?
+    @@system_test_utils.logout unless user_label.nil?
+  end
+
+  def complete_assessment_new_link(user_label, patient, assessment_label)
+    @@system_test_utils.login(user_label) unless user_label.nil?
+    visit "/r/#{patient.submission_token}/#{patient.jurisdiction.unique_identifier}/en/#{patient.initials}"
+    submit_assessment(ASSESSMENTS[assessment_label]['symptoms'])
+    err_msg = @@system_test_utils.get_err_msg('Monitoree assessment', 'submission message', 'existent')
+    assert page.has_content?('Thank You For Completing Your Self Report'), err_msg
+    @@assessment_form_verifier.verify_assessment(patient, ASSESSMENTS[assessment_label]) unless user_label.nil?
+    @@system_test_utils.logout unless user_label.nil?
+  end
+
+  def complete_assessment_old_link(user_label, patient, assessment_label)
```

Comment:
Thanks for adding tests for this for now. If we ever go back and remove those lookup tables, I think these can probably be removed at that time as well.

Reply:
yep!